pyproject.toml

```diff
@@ -5,6 +5,7 @@ requires = ["setuptools", "Cython", "numpy", "wheel"]
 [tool.setuptools]
 packages = [
   "meshpy",
+  "meshpy.FourC",
```

Comment:
I am not a fan of the spelling `FourC`, I would go with `four_c` to align with the spelling of other submodules in MeshPy.

Reply:
Also for  testing I think you have to add the submodule to the `pyproject.toml` file.

Reply:
Changed it to your suggestion of `four_c`, now the code is consistent with the remaining parts.

Maybe you could briefly elaborate what you mean by adding the submodule to the `pyproject.toml` file? Unfortunately, I do not understand it

Reply:
My bad, you already added the submodule correctly here.